ci/dra.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash -ie
+#Note - ensure that the -e flag is set to properly set the $? status if any command fails
+
+# Since we are using the system jruby, we need to make sure our jvm process
+# uses at least 1g of memory, If we don't do this we can get OOM issues when
+# installing gems. See https://github.com/elastic/logstash/issues/5179
+export JRUBY_OPTS="-J-Xmx1g"
+
+if [ -z "$VERSION_QUALIFIER_OPT" ]; then
+  RELEASE=1 rake artifact:all
+else
+  VERSION_QUALIFIER="$VERSION_QUALIFIER_OPT" RELEASE=1 rake artifact:all
+fi
+
+STACK_VERSION=`cat versions.yml | sed -n 's/^logstash\:\s\([[:digit:]]*\.[[:digit:]]*\.[[:digit:]]*\)$/\1/p'`
+
+echo "Creating dependencies report for ${STACK_VERSION}"
+mkdir -p build/reports/dependencies-reports/
+bin/dependencies-report --csv=build/reports/dependencies-reports/logstash-${STACK_VERSION}.csv
```

Comment:
I suggest at least printing a final success message, with some information about what was generated, for example:


```
❯ for file in build/logstash-*; do shasum $file;done  
63a4578a28bb88f461494cc1619eb6bb2dcb385a  build/logstash-8.5.0-aarch64.rpm
c019dbe419dd98cd04242da6d725488a55195ae9  build/logstash-8.5.0-amd64.deb
a183a0aa0343a06392c9a8914efa1a7a4b1a2f5a  build/logstash-8.5.0-arm64.deb
3ab631d1236bd34b5657a154727e6737c08d2209  build/logstash-8.5.0-darwin-aarch64.tar.gz
d8fb95b1de46c7246315d7bce183d1dba9d7264a  build/logstash-8.5.0-darwin-x86_64.tar.gz
c6775dfdaafdecc009ba7a9754d53f37539a9e09  build/logstash-8.5.0-linux-aarch64.tar.gz
e9d38879f76db4e314bec31e79bcd9c6aa92d810  build/logstash-8.5.0-linux-x86_64.tar.gz
c03f84a1901d5980464a4b25cd8588f96322b9d5  build/logstash-8.5.0-no-jdk.deb
d5f176e24ac60c2d51209a9a2107c5ce71c0ea8c  build/logstash-8.5.0-no-jdk.rpm
d4d4c572f16591255ef6ec7698e45253f227c340  build/logstash-8.5.0-no-jdk.tar.gz
b8793b727ffda82a9556e9d4d2ae94f735a6ea85  build/logstash-8.5.0-no-jdk.zip
4985c4e58baec53d1c184652002d5a8f420fa5fa  build/logstash-8.5.0-windows-x86_64.zip
36f18c9de1c61230ba0a36779c8da98a317dcca0  build/logstash-8.5.0-x86_64.rpm
c50b7232ad553571c90ce5185c8515372df07319  build/logstash-ironbank-8.5.0-docker-build-context.tar.gz
35a66975ef01087ff31a680ee8ae0ed8485d2d65  build/logstash-oss-8.5.0-aarch64.rpm
b9fc21c0efcccc0f53a19de1cfaefc9a6b4eccb5  build/logstash-oss-8.5.0-amd64.deb
85dbc0e37824e7ea90373429c2a981f0c0804222  build/logstash-oss-8.5.0-arm64.deb
35b88312fcf1410cec8bea0433fdbf4ef2a8854e  build/logstash-oss-8.5.0-darwin-aarch64.tar.gz
fc31ab1ff6c48ad31af09d261d60bb44c0bac84d  build/logstash-oss-8.5.0-darwin-x86_64.tar.gz
1d7f5dacd1388214cd28f539d1c6d301a37ac76b  build/logstash-oss-8.5.0-linux-aarch64.tar.gz
3c55047b5d32ca581e4adc4fd22ce69c53b0c4df  build/logstash-oss-8.5.0-linux-x86_64.tar.gz
5a0fc122b1ae7f6a5e3c180f4947940172778d7a  build/logstash-oss-8.5.0-no-jdk.deb
0c7583fdad688e96d076ad78035f343c4c2f63ab  build/logstash-oss-8.5.0-no-jdk.rpm
6e17613ef26c1399471869a8f8ccadd5c1368581  build/logstash-oss-8.5.0-no-jdk.tar.gz
5f17c0bc4a58c9cd88797f134788d8151607d6a1  build/logstash-oss-8.5.0-no-jdk.zip
1d195e45ce871ca3363d3eb3144ad89c76807cec  build/logstash-oss-8.5.0-windows-x86_64.zip
abc6a0b8c008e404ede34b26503f675ff12818e4  build/logstash-oss-8.5.0-x86_64.rpm
```